src/features/network-modifications/twoWindingsTransformer/tapChanger/TapChangerSteps.tsx

```
@@ -0,0 +1,324 @@
+        name: `${tapChanger}.${FieldConstants.STEPS}`,
+    });
+
+    const areStepsModified = useMemo(() => {
+        if (editData?.[FieldConstants.STEPS] && isNodeBuilt) {
+            return true;
+        } else {
+            return !compareStepsWithPreviousValues(
+                tapStepsWatcher,
+                toTapChangerStepList(previousValues?.[FieldConstants.STEPS])
+            );
+        }
```

ESLint

[error] Unnecessary 'else' after 'return'.

(no-else-return)